.Xamarin.Forms.iOS.slnf

```diff
@@ -23,12 +19,10 @@
       "Xamarin.Forms.Maps\\Xamarin.Forms.Maps.csproj",
       "Xamarin.Forms.Material.iOS\\Xamarin.Forms.Material.iOS.csproj",
       "Xamarin.Forms.Pages.Azure\\Xamarin.Forms.Pages.Azure.csproj",
-      "Xamarin.Forms.Pages.UnitTests\\Xamarin.Forms.Pages.UnitTests.csproj",
       "Xamarin.Forms.Pages\\Xamarin.Forms.Pages.csproj",
+      "Xamarin.Forms.Platform.iOS.UnitTests\\Xamarin.Forms.Platform.iOS.UnitTests.csproj",
```

Comment:
Are these changes necessary?

Reply:
Yes, I'm tired of having to add that project back into the solution filter every time I want to build.